client/src/index.html

```diff
@@ -6,6 +6,9 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script>
+      var global = global || window;
```

Comment:
Just curious: why is this script needed?

Reply:
It's needed for one of the libraries we used. Added it more as a fix to an error